libs/tdigest/src/test/java/org/elasticsearch/tdigest/TDigestTests.java

```diff
@@ -542,4 +544,8 @@ public void testMonotonicity() {
             lastQuantile = q;
         }
     }
+
+    protected static TDigestArrays arrays() {
+        return WrapperTDigestArrays.INSTANCE;
```

Comment:
In the next PR, I'll change this to the real BigArrays implementation

Reply:
It'd be nice, at least in tests, to track the unclosed stuff. I think you could try using `LeakTracker`.

Reply:
But that can wait for another PR if you want.